composer.json

```
@@ -22,22 +22,22 @@
     "require": {
         "php": "^7.2",
 
-        "symfony/config": "^3.4|^4.3",
-        "symfony/dependency-injection": "^3.4|^4.3",
-        "symfony/framework-bundle": "^3.4|^4.3",
-        "symfony/http-kernel": "^3.4|^4.3",
+        "symfony/config": "^4.3",
```

Shouldn't we allow for 5.0?
```suggestion
        "symfony/config": "^4.3|^5.0",
```

Also, do we want to remove support for 4.3?